underfs/s3/pom.xml

```diff
@@ -30,47 +30,45 @@
   </properties>
 
   <dependencies>
+    <!-- External dependencies -->
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-api</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-common</artifactId>
-      <version>${project.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>net.java.dev.jets3t</groupId>
-      <artifactId>jets3t</artifactId>
-    </dependency>
-
-    <!-- The following are jets3t dependencies
-     (see http://mvnrepository.com/artifact/net.java.dev.jets3t/jets3t/0.9.4).
-     We need to explicitly include them, because jets3t is shaded. -->
-    <!-- TODO(jsimsa): Investigate whether the same outcome can be achieved
-     using the promoteTransitiveDependencies option of the maven shade plugin. -->
-    <dependency>
+      <!-- jets3t dependency that needs to included explicitly, because jets3t is shaded. -->
       <groupId>com.jamesmurty.utils</groupId>
       <artifactId>java-xmlbuilder</artifactId>
-      <version>0.4</version>
     </dependency>
     <dependency>
+      <!-- jets3t dependency that needs to included explicitly, because jets3t is shaded. -->
```

Comment:
so that dependencies can be moved around or removed without losing this information